wiki/Disambiguation/osu!/en.md

```diff
@@ -3,5 +3,5 @@
 **osu!** can have multiple meanings:
 
 - The game and its community
-- The default [game mode](/wiki/Game_mode), sometimes referred to as *[osu!standard](/wiki/Game_mode/osu!)*
+- The default [game mode](/wiki/Game_mode), previously known as *[osu!standard](/wiki/Game_mode/osu!)*
```

Comment:
has it really ever been known as such? think it's best if you remove everything after comma:

```suggestion
- The [default game mode](/wiki/Game_mode/osu!)
```